src/main/java/duke/UI.java

```diff
@@ -0,0 +1,101 @@
+package duke;
+
+/**
+ * UI class to abstract all UI related printing.
+ */
+public class UI {
+
+    /**
+     * Commonly used beautifying line.
+     */
+    public static String line = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+
+    /**
+     * Prints a single line.
+     */
+    static void println() {
+        System.out.println(line);
+    }
+
+    /**
+     * Prints string s wrapped in lines.
+     *
+     * @param s s
+     */
+    static void say(String s) {
```

Comment:
I like the UI class has a lot of clear helper functions and that the functions that print outputs are very clearly written.